src/main/kotlin/com/evalify/evalifybackend/user/service/UserService.kt

```
@@ -1,20 +1,202 @@
+                name = request.name,
+                email = request.email,
+                password = request.password,
+                role = request.role,
+                phoneNumber = request.phoneNumber,
+                isActive = request.isActive,
+                createdAt = Timestamp.from(Instant.now())
+            )
+
+            val savedUser = userRepository.save(user)
+            return savedUser.toUserResponse()
+        } catch (e: IllegalArgumentException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)